vispy/scene/cameras/cameras.py

```diff
@@ -85,7 +76,8 @@ class BaseCamera(Node):
     # These define the state of the camera
     _state_props = ()
 
-    ZOOM_FACTOR = 0.005
+    # The fractional zoom to apply for a single pixel of mouse motion 
+    zoom_factor = 0.007
```

Comment:
I'm fine with either one. I'm also okay with tweaking it in a subsequent PR where we take a mini poll :)